users/views.py

```diff
@@ -181,4 +185,80 @@ def post(self, request):
             return JsonResponse({'MESSAGE':False}, status=200)
 
         except KeyError:
-            return JsonResponse({'MESSAGE':'KEY_ERROR'}, status=400)
\ No newline at end of file
+            return JsonResponse({'MESSAGE':'KEY_ERROR'}, status=400)
+
+class ReviewView(View):
+    @login_required
+    def post(self,request,product_id):
```

Comment:
```suggestion
    def post(self,request,product_id=None):
```
보통 그럴일은 없겠지만, product_id를 받으면 그부분에 대한 검증도 해줘야 합니다. 또 인자가 전달이 안되면 500 에러가 날텐데  이걸 방지하려면  디폴트 값을 주세요.

Reply:
아래 두 메소드도 같습니다. 세개만 해결해주세요. 로직은 많이 깔끔해졌네요~👍

Reply:
수정 완료 했습니다~